tests/storage/reservation.go

```diff
@@ -360,6 +362,42 @@ var _ = Describe(SIG("SCSI persistent reservation", Serial, func() {
 			By("Ensuring all KubeVirt components are ready")
 			testsuite.EnsureKubevirtReady()
 		})
+
+		Context("With multipath", func() {
+			var mpathCleanUp bool
+			const mpathSocket = "/proc/1/root/run/multipathd.socket"
+			BeforeEach(func() {
+				// Check if mulitpathd socket exists on the nodes, if not simulate the existance by creating a mock socket
+				nodes := libnode.GetAllSchedulableNodes(virtClient)
+				for _, node := range nodes.Items {
+					_, err := libnode.ExecuteCommandInVirtHandlerPod(node.Name, []string{"ls", mpathSocket})
+					if err != nil {
+						By(fmt.Sprintf("Create a fake mulitpathd.socket in node %s", node.Name))
+						libnode.ExecuteCommandInVirtHandlerPod(node.Name, []string{"touch", mpathSocket})
+						mpathCleanUp = true
+					}
+				}
+			})
+
+			AfterEach(func() {
+				if mpathCleanUp {
+					nodes := libnode.GetAllSchedulableNodes(virtClient)
+					for _, node := range nodes.Items {
+						_, err := libnode.ExecuteCommandInVirtHandlerPod(node.Name, []string{"rm", "-f", mpathSocket})
+						Expect(err).ToNot(HaveOccurred())
+					}
+				}
+			})
+			It("ensure multipath socket is bind mounted and available to the pr-helper daemon", func() {
+				nodes := libnode.GetAllSchedulableNodes(virtClient)
+				for _, node := range nodes.Items {
+					output, err := libnode.ExecuteCommandInVirtHandlerPod(node.Name, []string{"ls",
```

Comment:
Would it make sense to check `/proc/mounts` if bind mount is actually in place? IIRC then the multipath watcher needs to create the target file before creating the bind mount. That means the file could exist while there no bind mount.

Reply:
yes, we can let me change this.